Packs/Anomali_ThreatStream/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Anomali ThreatStream v2
+- Fixed an issue where the *tags* data was missing at the output of certain commands.
```

Comment:
```suggestion
- Fixed an issue where the *tags* information was missing from the context outputs of certain commands.
```

If I am getting this right you only added tags to 4 commands - please name them in the RN.